ocs_ci/helpers/dr_helpers.py

```diff
@@ -997,6 +1026,14 @@ def wait_for_replication_resources_deletion(
                 should_exist=False,
             )
 
+        if is_cg_cephfs_enabled():
```

Comment:
In 4.20, CG will be enabled for both RBD and CephFS, can we reuse the same `is_cg_enabled`, or do we need a separate CG CephFS check?